openupgrade_scripts/scripts/account/17.0.1.2/upgrade_analysis_work.txt

```diff
@@ -0,0 +1,483 @@
+NEW ir.actions.server: account.action_automatic_entry_change_account
+NEW ir.actions.server: account.action_automatic_entry_change_period
+NEW ir.actions.server: account.action_move_switch_move_type
+DEL ir.actions.server: account.action_automatic_entry
+DEL ir.actions.server: account.action_move_switch_invoice_to_credit_note
+NEW ir.cron: account.ir_cron_account_move_send
+NEW ir.model.access: account.access_account_journal_group_invoice
+NEW ir.model.access: account.access_account_journal_group_readonly
+NEW ir.model.access: account.access_account_move_send
+DEL ir.model.access: account.access_account_account_template
+DEL ir.model.access: account.access_account_chart_template
+DEL ir.model.access: account.access_account_fiscal_position_account_template
+DEL ir.model.access: account.access_account_fiscal_position_tax_template
+DEL ir.model.access: account.access_account_fiscal_position_template
+DEL ir.model.access: account.access_account_group_template
+DEL ir.model.access: account.access_account_invoice_send
+DEL ir.model.access: account.access_account_journal_group_all
+DEL ir.model.access: account.access_account_reconcile_model_line_template
+DEL ir.model.access: account.access_account_reconcile_model_template
+DEL ir.model.access: account.access_account_tag_internal_user
+DEL ir.model.access: account.access_account_tax_repartition_line_template_manager
+DEL ir.model.access: account.access_account_tax_template
+DEL ir.model.access: account.access_product_template_account_manager
+DEL ir.model.constraint: account.constraint_account_account_code_company_uniq
+DEL ir.model.constraint: account.constraint_account_fiscal_position_account_account_src_dest_uniq
+DEL ir.model.constraint: account.constraint_account_fiscal_position_tax_tax_src_dest_uniq
+DEL ir.model.constraint: account.constraint_account_group_check_length_prefix
+DEL ir.model.constraint: account.constraint_account_journal_code_company_uniq
+DEL ir.model.constraint: account.constraint_account_journal_group_uniq_name
+DEL ir.model.constraint: account.constraint_account_move_line_check_accountable_required_fields
+DEL ir.model.constraint: account.constraint_account_move_line_check_amount_currency_balance_sign
+DEL ir.model.constraint: account.constraint_account_move_line_check_credit_debit
+DEL ir.model.constraint: account.constraint_account_move_line_check_non_accountable_fields_null
+DEL ir.model.constraint: account.constraint_account_payment_check_amount_not_negative
+DEL ir.model.constraint: account.constraint_account_payment_method_name_code_unique
+DEL ir.model.constraint: account.constraint_account_reconcile_model_name_unique
+DEL ir.model.constraint: account.constraint_account_report_line_code_uniq
+DEL ir.model.constraint: account.constraint_account_tax_name_company_uniq
+DEL ir.model.constraint: account.constraint_account_tax_template_name_company_uniq
+DEL ir.model.constraint: account_sequence.constraint_account_move_unique_name
```

Comment:
this one is actually added in account now, so we need to uniquify names.

And the whole _work file needs updating from the current analysis file, it contains confusing bogus entries